src/python/pants/backend/python/util_rules/pex_from_targets.py

```diff
@@ -606,55 +636,29 @@ async def _setup_constraints_repository_pex(
 @dataclass(unsafe_hash=True)
 class RequirementsPexRequest:
     addresses: tuple[Address, ...]
-    internal_only: bool
```

Comment:
This was always `True` :|

Reply:
Yea, I think that that makes sense... essentially, the optimization to split out requirements from user code can only possibly be relevant for "internal" PEXes (those not published outside of Pants).

It would be good to add a docstring to `RequirementsPexRequest` that explains what it results in... roughly by spelling out the values used in `generalize_requirementspexrequest`: "Requests a PEX containing only thirdparty requirements for internal/non-portable use. Used as part of an optimization to ... (etc)".